frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -1,41 +1,92 @@
 import React, { useState, useEffect } from 'react';
-import { Item, Card } from 'semantic-ui-react';
+import { Form, Item, Card } from 'semantic-ui-react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import { Emitters } from '../../../utils';
 import { ShoutoutsAPI, Shoutout } from '../../../API/ShoutoutsAPI';
 import styles from './AdminShoutouts.module.css';
 
 const AdminShoutouts: React.FC = () => {
   const [shoutouts, setShoutouts] = useState<Shoutout[]>([]);
+  const [displayShoutouts, setDisplayShoutouts] = useState<Shoutout[]>([]);
   useEffect(() => {
-    ShoutoutsAPI.getAllShoutouts().then((shoutouts) => setShoutouts(shoutouts));
+    ShoutoutsAPI.getAllShoutouts().then((shoutouts) => {
+      setShoutouts(shoutouts);
+      setDisplayShoutouts(shoutouts);
+    });
   }, []);
+  const [earlyDate, setEarlyDate] = useState<Date>(new Date('2022/01/01'));
```

Comment:
I'm thinking it might make more sense to change this to today's date, since the most common use case is to display the shoutouts at all-hands, which were made the same day. So the "customizability" isn't needed until someone, in some less common scenario wants to display shoutouts for some other date range.

Reply:
I thought people submit shoutouts leading up to an all hands and during all hands?

Reply:
In more recent all-hands, I remember submitting shoutouts when the form link was posted, which was usually at the very beginning of all-hands. And my shoutouts usually are the first one, so I assume they aren't opening up shoutouts any earlier than the same day of all-hands.

If they do, they I guess this is fine as is. In that case, probably just want to make the default year dynamic to the current year.

Thoughts @JacksonStaniec?

Reply:
I agree with @andrew032011 that the date should be set dynamically; it'll just avoid potentially confusing bugs in the future. But @airaagrawal1 has the right idea about what we want to do with the IDOL shoutouts system, where people can submit shoutouts leading up to and during all-hands instead of just during it. The leads are probably going to want to display the shoutouts between the last all-hands and the current one, so maybe the last 2 weeks is a good default range?

Reply:
This sounds right to me.

Reply:
Sounds good 👍